diff --git a/node_modules/webdriverio/build/index.js b/node_modules/webdriverio/build/index.js
index 9cf6df9..1c14302 100644
--- a/node_modules/webdriverio/build/index.js
+++ b/node_modules/webdriverio/build/index.js
@@ -3352,11 +3352,11 @@ function getPolyfillManager(browser2) {
 }
 var log4 = logger4("webdriverio:PolyfillManager");
 var polyfillFn = function webdriverioPolyfill() {
-  const __defProp2 = Object.defineProperty;
-  const __name = function(target, value) {
-    return __defProp2(target, "name", { value, configurable: true });
+  var __defProp = Object.defineProperty;
+  var __name = function (target, value) {
+    return __defProp(target, "name", { value: value, configurable: true });
   };
-  const __globalThis = typeof globalThis === "object" && globalThis || typeof window === "object" && window;
+  var __globalThis = (typeof globalThis === "object" && globalThis) || (typeof window === "object" && window);
   __globalThis.__name = __name;
 };
 var _initialize, _browser4, _scriptsRegisteredInContexts, _PolyfillManager_instances, registerScripts_fn;
diff --git a/node_modules/webdriverio/build/node.js b/node_modules/webdriverio/build/node.js
index 6efec11..a420040 100644
--- a/node_modules/webdriverio/build/node.js
+++ b/node_modules/webdriverio/build/node.js
@@ -3535,11 +3535,11 @@ function getPolyfillManager(browser2) {
 }
 var log5 = logger5("webdriverio:PolyfillManager");
 var polyfillFn = function webdriverioPolyfill() {
-  const __defProp2 = Object.defineProperty;
-  const __name = function(target, value) {
-    return __defProp2(target, "name", { value, configurable: true });
+  var __defProp = Object.defineProperty;
+  var __name = function (target, value) {
+    return __defProp(target, "name", { value: value, configurable: true });
   };
-  const __globalThis = typeof globalThis === "object" && globalThis || typeof window === "object" && window;
+  var __globalThis = (typeof globalThis === "object" && globalThis) || (typeof window === "object" && window);
   __globalThis.__name = __name;
 };
 var PolyfillManager = class _PolyfillManager extends SessionManager {
